### PR TITLE
Fix registry url

### PR DESCRIPTION
**Pull request by gibchikafa**
### What changes were proposed in this pull request?

(Please outline the changes and how this PR fixes the issue.)

### Why are the changes needed?

(Please clarify why the changes are needed. For instance,
  1. If you propose a new API, clarify the use case for a new API.
  2. If you fix a bug, describe the bug.)

Fix: # (issue)

### Does this PR introduce _any_ user-facing change?

(Please list the user-facing changes introduced by your change, including
  1. Change in user-facing APIs.
  2. Addition or removal of property keys.)

No.

### How was this patch tested?

(Please test your changes, and provide instructions on how to test it:
  1. If you add a feature or fix a bug, add a test to cover your changes. 
  2. If you fix a flaky test, repeat it for many times to prove it works.)

### CODE REVIEW

File: deploy/kubernetes/docker/build.sh [hunk 1]
```diff
@@ -138,7 +138,6 @@ cp "$RSS_DIR/$RSS_FILE" .
 GIT_BRANCH=$(git rev-parse --abbrev-ref HEAD)
 GIT_COMMIT=$(git describe --dirty --always --tags | sed 's/-/./g')
 IMAGE_VERSION=$(cat $RSS_DIR/version.txt | awk '{$1=$1; print}' | sed '/^$/d')
-IMAGE=$REGISTRY/rss-server:$IMAGE_VERSION
```
Comment (maismail) [inline deploy/kubernetes/docker/build.sh:141]: why remove it?

Comment (gibchikafa) [reply deploy/kubernetes/docker/build.sh:141]: It is redudant. Being redeclared a line after

Comment (maismail) [reply deploy/kubernetes/docker/build.sh:141]: I might be missing it, but seems there is no other definition

Comment (gibchikafa) [reply deploy/kubernetes/docker/build.sh:141]: Here https://github.com/gibchikafa/incubator-uniffle/blob/af9078f90f39bad51d5b10e2c88daa4940a6b6c7/deploy/kubernetes/docker/build.sh#L142

**Review by maismail — APPROVED** [review]

**Pull request merged**